### PR TITLE
feature/CORE-605/create-contact-property-from-flow-builder

### PR DESCRIPTION
**Pull request by bulatgab**
1. Adding a new 'create contact property' option to GenericContactPropertyEditor + event emission, so that we can show a modal where the user can specify a new contact property and make a request to the server.
2. Suggestions to improve appConfig & builderConfig types.

### CODE REVIEW

File: src/builder.config.json
Comment (bulatgab) [inline src/builder.config.json:1]: Not sure why it is shown as new, the idea was to move it from root folder to `src`

File: src/app.config.json
Comment (bulatgab) [inline src/app.config.json:1]: Not sure why it is shown as new, the idea was to move it from root folder to `src`

File: dist/types/builder.config.json
Comment (bulatgab) [inline dist/types/builder.config.json:1]: there it is - 1

File: dist/types/app.config.json
Comment (bulatgab) [inline dist/types/app.config.json:1]: there it is - 2

**Review by object-required — APPROVED** [review]

File: src/app.config.json
Comment (safydy) [reply src/app.config.json:1]: thanks for the solution, let's pair tmr to fix the CI build ;)

**Review by safydy — CHANGES_REQUESTED** [review]
Let's pair to fix the CI build tmr

File: src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue
Comment (bulatgab) [inline src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue:1]: these changes were not necessary, but I kept them

**Pull request merged**